src/server.js

```diff
@@ -139,6 +139,35 @@ function init(socket, client) {
 	if (!client) {
 		socket.emit("auth", {success: true});
 		socket.on("auth", auth);
+		socket.on("sign-up", (data) => {
```

Comment:
This event should only be bound if sign up is enabled in the config, otherwise it's technically possible to create users.